src/app/cube/details/details.component.ts

```diff
@@ -167,32 +167,22 @@ export class DetailsComponent implements OnInit, OnDestroy {
           this.setAcademicLevels();
           this.setLearningObjectAuthors();
           this.loading = false;
-        }
-        this.learningObjectService.fetchLearningObjectWithResources(
```

Comment:
This is deleted because Learning Object Service is now updated to send back the revision URI